crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -94,7 +97,13 @@ pub async fn create(
     let mut output = vec![("address", format!("{address:#x}"))];
     if account_json["deployed"] == json!(false) {
         println!("Account successfully created. Prefund generated address with at least {max_fee} tokens. It is good to send more in the case of higher demand, max_fee * 2 = {}", max_fee * 2);
-        output.push(("max_fee", format!("{max_fee:#x}")));
+        // For default and Int it should be in int
+        let max_fee_str = if value_format == ValueFormat::Hex {
+            format!("{max_fee:#x}")
+        } else {
+            format!("{max_fee:#}")
+        };
```

Comment:
Could we maybe have some trait for ValueFormat, so it would be possible to call `value_format.format(max_fee)`?

Reply:
don't think a trait is required, directly added a method on the type.